### DISCUSSION TITLE
How do I find the logs?

### DISCUSSION THREAD

**Discussion by begunfx** [haugene/docker-transmission-openvpn#2616] [Q&A] [answered]
I posted an issue, but am unable to include the logs because when I check the log inside the Docker Container (in Synology) it's always empty.  If I'm supposed to see something there, then perhaps I need to set an option with the Docker run command?  Otherwise, do the logs normally live anywhere else?

Thanks in advance for your help!

**Comment by pkishino** [comment]
Had a quick search.. recommend setting up portainer container as dsm is very restrictive.. then easily manage containers etc

**Reply by begunfx** [reply]
Okay.  Thanks @pkishino.  I actually create the docker containers using the task scheduler and the docker run command.  It's convienent, more configurable and if I need to re-run a docker command very simple.  So if I understand you correctly I should be seeing the log insider the docker container itself then.  Correct?

**Reply by pkishino** [reply, marked as answer]
No, the logs are produced by the container and docker application ownes them. So you need to use either the cli or e.g portainer as the synology gui is very limited

**Reply by begunfx** [reply]
Okay.  Sounds good.  I installed portainer and it seems I can now see logs.  I'll try to rebuild the container so I can capture the logs with the issue I had posted.  Thanks!
